sql/wsrep_server_service.cc

```diff
@@ -107,7 +140,10 @@ void Wsrep_server_service::release_high_priority_service(wsrep::high_priority_se
     static_cast<Wsrep_high_priority_service*>(high_priority_service);
   THD* thd= hps->m_thd;
   delete hps;
+  wsrep_store_threadvars(thd);
+  thd->set_mysys_var(0);
```

Comment:
why is clearing `mysys_var` necessary?
`thd` is being deleted and I didn't find it being referenced in the destructor (I could be wrong)